### PR TITLE
Minor Fixes 

### PR DESCRIPTION
**Pull request by jcharkow**
# Description

Fix import errors that occur with recent version of scipy 

Fix typos

Fix deprecation warnings 

# Contents ([#134](https://github.com/Roestlab/massdash/pull/134))

### Other
- deprecation in latest scipy
- deprecation  (replace warn --> warning)
- typo

### Uncategorised!
- update scipy version in configs

### CODE REVIEW

**Comment by jcharkow** [comment]
@singjc ideally we can do a quick release with these changes like version 0.91 for these minor changes. Currently tests fail because of recent scipy bump

**Comment by singjc** [comment]
> @singjc ideally we can do a quick release with these changes like version 0.91 for these minor changes. Currently tests fail because of recent scipy bump

Yeah think we can do this. There is a github action that you have to trigger manually to bump the version. But I think the script currently bumps the patch version by a whole number.

File: massdash/plotting/InteractivePlotter.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 # Data modules
 import numpy as np
-from scipy.signal import savgol_filter, gaussian, convolve
+from scipy.signal import savgol_filter, convolve
+from scipy.signal.windows import gaussian
```
Comment (singjc) [inline massdash/plotting/InteractivePlotter.py:12]: Can you add the specific version dependency for scipy in requirements, setup.cfg and project.toml?

**Review by singjc — APPROVED** [review]
Looks good. Can you add the specific version constraint for scipy in the requirements, setup.cfg and project.toml.

**Comment by singjc** [comment]
> > @singjc ideally we can do a quick release with these changes like version 0.91 for these minor changes. Currently tests fail because of recent scipy bump
> 
> Yeah think we can do this. There is a github action that you have to trigger manually to bump the version. But I think the script currently bumps the patch version by a whole number.

Actually, I think it's okay if we bump by whole number,, because it's just a patch. So v0.010, otherwise it would be first initial version with 91 patches?

**Comment by jcharkow** [comment]
sounds good we can do version 0.0.10

**Pull request merged**